apps/main/src/llamalend/features/user-position-history/cells/CollateralChangeCell.tsx

```diff
@@ -0,0 +1,30 @@
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import type { CellContext } from '@tanstack/react-table'
+import { formatNumber } from '@ui/utils'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import type { ParsedUserLendCollateralEvent } from '../hooks/useUserLendCollateralEvents'
+
+const { Spacing } = SizesAndSpaces
+
+export const CollateralChangeCell = ({ row }: CellContext<ParsedUserLendCollateralEvent, any>) => (
```

Comment:
It's maybe a matter of taste, but I find `row.original` a bit hard to read.

```suggestion
export const CollateralChangeCell = ({ row: { original: { collateralChange } } }: CellContext<ParsedUserLendCollateralEvent, any>) => (
```
```suggestion
export const CollateralChangeCell = ({ row: { original: collateralEvent } }: CellContext<ParsedUserLendCollateralEvent, any>) => (
```

Reply:
Destructing is good taste imo, will apply